modules/services/cliphist.nix

```
@@ -17,6 +17,14 @@ in {
       '';
     };
 
+    flags = lib.mkOption {
```

This type of option is typically called `extraOptions`. If you search, you'll find multiple examples so I would suggest to follow that convention:
```suggestion
    extraOptions = lib.mkOption {
```

---

Huh, I didn't know that yet, and remembered adding the flag option to atuin some time ago. Thanks for sharing.

---

No worries, it's not a hard rule or anything.